tilings/strategies/sliding.py

```diff
@@ -0,0 +1,276 @@
+from typing import Callable, DefaultDict, Dict, Iterator, List, Optional, Tuple
+
+from comb_spec_searcher.strategies.strategy import (
+    DisjointUnionStrategy,
+    StrategyFactory,
+)
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import (
+    get_col_info,
+    gp_slide,
+    gp_slide_inverse,
+    slidable_pairs,
+    slide_assumption,
+    slide_column,
+)
+
+
+def _tiling_identity_map(tiling: Tiling) -> Tiling:
+    return tiling
+
+
+def _tiling_reverse(tiling: Tiling) -> Tiling:
+    t: Tiling = tiling.reverse()
+    return t
+
+
+def _tiling_rotate90(tiling: Tiling) -> Tiling:
+    t: Tiling = tiling.rotate90()
+    return t
+
+
+def _tiling_rotate90_inverse(tiling: Tiling) -> Tiling:
+    t: Tiling = tiling.rotate270()
+    return t
+
+
+def _tiling_rotate90_and_reverse(tiling: Tiling) -> Tiling:
+    t: Tiling = tiling.rotate90().reverse()
+    return t
+
+
+def _tiling_rotate90_and_reverse_inverse(tiling: Tiling) -> Tiling:
+    t: Tiling = tiling.reverse().rotate270()
+    return t
```

Comment:
Do these really need functions?

Reply:
Even the case where you apply two symmetries could be a single application of one symmetry - i.e. the anti-diagonal or complement symmetries (I think).